src/request-sender.ts

```diff
@@ -10,22 +11,33 @@ import Response from './response';
 import Timeout from './timeout';
 
 export default class RequestSender {
+    private _cache: Cache;
```

Comment:
🍹 should this be `readonly` ? I dont see any mechanisms to set a new cache object once RequestSender has been instantiated